src/Components/FeelingRoom/FeelingRoom.js

```diff
@@ -1,11 +1,82 @@
 import React from 'react';
 import './FeelingRoom.scss';
+import {Component} from 'react'
 
-const FeelingRoom = () => {
-
-  return (
-    <h1>Feeling Room</h1>
-  )
+class FeelingRoom extends Component {
```

Comment:
This is a great starting point to get the different mood based themes going in the feelingRoom